docs/EN/GettingHelp/GeneralTroubleshooting.md

```diff
@@ -30,47 +36,43 @@ Ignore it: More details, Install anyway.
 
 * [Nightscout Client](../GettingHelp/TroubleshootingNsClient.md)
 
-### Usage
+### **Usage**
 * [Wrong carb values](#CobCalculation-detection-of-wrong-cob-values)
 
    ![Error: Slow carb absorption](../images/Calculator_SlowCarbAbsorption.png)
 
 * [SMS commands](#SMSCommands-troubleshooting)
 
-### Cannot start Omnipod with Android 16
-
-Upgrade to minimum version of AndroidAPS: 3.3.2.1.
-
-### Frequent Bluetooth connection problems
+---
 
-#### Android 15
+(generaltroubleshooting-bluetooth-related-issues)=
 
-After upgrading Android or moving to a recent phone, **AAPS** frequently loses Bluetooth connection to the pump. The problem disappears temporarily when restarting the phone. If the phone runs Android 15, you can try to enable the following:
 
-1) **Open preferences** by clicking the three-dot menu on the top right side of the home screen.
+## **Bluetooth related issues**
 
+For known issues with Bluetooth connections, dropouts of pump/pods, or activation and connection issues [Bluetooth Troubleshooting](../GettingHelp/BluetoothTroubleshooting.md)
 
-![Open preferences](../images/Pref2020_Open2.png)
+---
 
-2. Scroll down and open the **Confirmation beeps** / **Advanced** submenu. Enable **Bond BT device on Android 15+**.
+(generaltroubleshooting-android-related-issues)=
 
-   ![BondBT](../images/troubleshooting/BondBT.png)
+## **Android Related Issues**
 
-3. If the pump asks for a pairing request, accept it.
+### **Battery optimization**
 
-4. Restart your phone.
+Android has implemented battery saving setting that are enabled by default. These settings automatically suspend/pause applications that are not required for the system to function to help conserve the amount of battery energy used by apps that don't always need to be running.
 
-#### Battery optimization
+When this is enabled, it will very likely cause issue for **AAPS** and other supporting apps like **xDrip+**.
 
-This can happen with various pumps. Apart from excluding AAPS from any battery optimization, you can also exclude the system Bluetooth app from battery optimization. This can help in some cases.
-Depending on the phone you use, you will find the Bluetooth app differently. 
+It's important to ensure that you have disabled Battery Optimization to ensure **AAPS** and other supporting apps remain active all the time.
 
-Here are examples how to find them on specific android phones.
+Depending on your phone model and make there may be more than one location and setting which needs to have this disabled.
 
+***NOTE:** Follow the steps below to Disable Battery Optimization for the Bluetooth service if your phone has this option, the same steps can be used to disable for **AAPS** and other apps, however the screenshots will only show how to do this for the Bluetooth service.*
```

Comment:
"the same steps can be used to disable for **AAPS** and other apps" not sure of the grammar.
Maybe make it an admonition?
`
```{admonition} Disable Battery Optimization
:class: note

Follow the steps below to Disable Battery Optimization for the Bluetooth service if your phone has this option, the same steps can be used to disable it for **AAPS** and other apps, however the screenshots will only show how to do this for the Bluetooth service.
```
`